### PR TITLE
HBase side filtering

### PR DESCRIPTION
**Pull request by kgrech**
Ability to set HBase side filters. Related to issue #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,13 @@ Finally, there is a lower level access to the raw `org.apache.hadoop.hbase.clien
     val table = "t1"
     val rdd = sc.hbase(table)
 
+HBase side filter are also supported by providing a custom Filter or Scan object:
```
Comment (vidma) [inline README.md]: filter**s** are

**Comment by kgrech** [comment]
@vidma I've fixed README file typo

**Comment by kgrech** [comment]
Hey, I have added one more commit. It is the same functionality, but much better implementation. Now I am simply overloading the functions instead of duplication of the existed code.

**Comment by andreaferretti** [comment]
Thank you for your contribution! It is great to see that people are interested in this project and willing to help!

We are in the process of implementing support to read and write timestamps. We plan to have this ready in a few days, and then we can merge your pull request to have support for server-side filters and publish a new version.

**Pull request merged**